charts/govuk-apps-conf/templates/router-configmap.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: router-nginx-conf
+  labels:
+    {{- include "govuk-apps-conf.labels" . | nindent 4 }}
+    app.kubernetes.io/component: web
+data:
+  nginx.conf: |-
```

Comment:
It would be good to document (perhaps in your commit message) why this config was chosen. With config like this it's often hard to figure out if there was an important reason particular config was added, or if it is safe to remove. 

Documenting the intent behind Nginx config will make it much easier to avoid 'teleological' config where we explain it by how it behaves rather than how it is designed to behave.

For instance, were these config options (like lowercasing urls) taken from the Puppet config, if so maybe we can port over the reasons given there?